### PR TITLE
feat: #7 add vp-server

### LINKED ISSUE #7: Add vaccination-pass-server to infrastructure repo

### PR DESCRIPTION
**Pull request by n-krueger**
Closes #7 

### TODO:
- [x] change image tag to `develop` once https://github.com/FHIR-Vaccination-Pass/vaccination-pass-server/pull/23 is merged
- [x] add auto-bootstrapping configuration once https://github.com/FHIR-Vaccination-Pass/vaccination-pass-server/pull/25 is merged
- [x] add README for how to run

### CODE REVIEW

File: docker-compose.yaml [hunk 1]
```diff
@@ -31,5 +47,33 @@ services:
...
+      KC_HEALTH_ENABLED: true
       KEYCLOAK_ADMIN: admin
       KEYCLOAK_ADMIN_PASSWORD: admin
+    restart: on-failure
+    healthcheck:
+      test: ['CMD', 'curl', '-k', '-f', 'https://localhost:10443/q/health/ready']
+      start_period: 30s
+      interval: 10s
+
+  vp-server:
```
Comment (tklausen) [inline docker-compose.yaml:59]: Can we add an option to choose whether the vp-server is started? Maybe, [Compose Profiles](https://docs.docker.com/compose/profiles/) are suitable for this.

**Comment by tklausen** [comment]
I think it would also be good to include a readme here. It should describe how to generate the certificates and how to configure whether the vp-server should start.

File: docker-compose.yaml (hunk 1 above)
Comment (n-krueger) [reply docker-compose.yaml:59]: I think profiles are the opposite functionality of what we want. By default you'd want everything to run. You can run `docker compose up ibm-fhir keycloak` to only run those services.

Comment (tklausen) [reply docker-compose.yaml:59]: Right, I totally forgot about that. Sorry.

**Review by tklausen — APPROVED** [review]
Nice work!

**Pull request merged**